lib/Ed25519VerificationKey2020.js

```diff
@@ -234,6 +284,59 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
     return exportedKey;
   }
 
+  /**
+   * Returns the JWK representation of this key pair.
+   *
+   * @see https://datatracker.ietf.org/doc/html/rfc8037
+   *
+   * @param {object} [options={}] - Options hashmap.
+   * @param {boolean} [options.publicKey] - Include public key?
+   * @param {boolean} [options.privateKey] - Include private key?
+   *
+   * @returns {{kty: string, crv: string, x: string, d: string}} JWK
+   *   representation.
+   */
+  toJwk({publicKey = false, privateKey = false} = {}) {
```

Comment:
Seems like `publicKey` should default to `true`, not `false`. It would also be pretty weird to set both of these to `false` and get a JWK with no key material -- so maybe add an exception for that case.

Reply:
@dlongley ah, good point. thanks